engine/src/main/java/org/hibernate/search/spi/SearchFactoryBuilder.java

```diff
@@ -1,40 +0,0 @@
-/*
- * Hibernate Search, full-text search for your domain model
- *
- * License: GNU Lesser General Public License (LGPL), version 2.1 or later
- * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
- */
-
-package org.hibernate.search.spi;
-
-import org.hibernate.search.cfg.spi.SearchConfiguration;
-import org.hibernate.search.engine.integration.impl.ExtendedSearchIntegrator;
-
-/**
- * @deprecated Use SearchIntegrationBuilder instead. This class will be removed!
- */
-@Deprecated
-public class SearchFactoryBuilder extends SearchIntegratorBuilder {
```

Comment:
I don't know if it was related, but maybe you could also remove the deprecated `SearchFactoryIntegrator`?